resources/views/dashboard/controllers/profile.blade.php

```diff
@@ -1,178 +1,235 @@
 @extends('layouts.dashboard')
```

Comment:
As a note of caution - please do not allow your personal auto-formatter to arbitrarily indent all of the code. This makes it very difficult to review... can't easily tell what was legitimately changed.

Reply:
Reverted